.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - '6'
-  - '7'
+  - '8'
+  - '9'
```

Comment:
10

Reply:
Just `10` or `8` & `10` ?

Reply:
@atian25

Reply:
don't need to modify this file manually, just update `package.json` then `node ./node_modules/egg-ci/install.js`